docs/2.6.5/docs/daml/intro/1_Token.rst

```diff
@@ -31,7 +31,7 @@ Individual contracts are *immutable* in the sense that an active contract can no
 
 Daml specifies what transactions are legal on a Daml Ledger. The rules the Daml code specifies are collectively called a *Daml model* or *contract model*.
 
-Daml Files and Modules
+Daml Modules and Files
```

Comment:
This page probably needs a bit of an editing pass as I'm now realizing this is not the only discrepancy between the list of titles and the actual titles, but this is the one I had noticed.

Reply:
Thanks for the heads-up -- will do that in a separate PR